iree/turbine/aot/compiled_module.py

```diff
@@ -568,6 +627,22 @@ def save_mlir(inst: "CompiledModule", path: Union[Path, str]):
 
     jittable = staticmethod(builtins.jittable)
 
+    @staticmethod
+    def annotate(
```

Comment:
Name this something better than `annotate`. Even `signature_info` would be sufficient to explain that its adding additional information to the function signature.